core/src/main/java/org/jruby/truffle/nodes/core/BignumNodes.java

```diff
@@ -192,7 +192,7 @@ public RubyBignum pow(RubyBignum a, RubyBignum b) {
 
     }
 
-    @CoreMethod(names = "/", required = 1)
+    @CoreMethod(names = {"/", "__slash__"}, required = 1)
```

Comment:
Probably their way to not be polluted by `mathn` which redefines `/`.
I wonder why `div` or `quo` is not sufficient though.